src/Business Foundation/Test/NoSeries/src/NoSeriesTests.Codeunit.al

```diff
@@ -908,6 +909,54 @@ codeunit 134530 "No. Series Tests"
         LibraryAssert.AreEqual('7', NoSeries.GetNextNo(TempNoSeriesLine, WorkDate()), 'GetNextNo with temporary record returned wrong value');
     end;
 
+    [Test]
+    procedure TestNoSeriesEmptyCodeInLine()
```

Comment:
it would be good to add the same tests on the old/legacy code to ensure we get the same behaviour as before where possible...